charts/piraeus/values.yaml

```diff
@@ -82,3 +82,10 @@ operator:
     kernelModuleInjectionImage: quay.io/piraeusdatastore/drbd9-bionic:v9.0.25
     kernelModuleInjectionMode: Compile
     kernelModuleInjectionResources: {}
+HAController:
```

Comment:
Helm values should begin with a lowercase letter https://helm.sh/docs/chart_best_practices/values/